testdata/p4_16_samples_outputs/stack2.p4

```diff
@@ -0,0 +1,16 @@
+#include <core.p4>
```

Comment:
In principle, this should be possible, but it would be a deeper change based on how the passes are currently organized:

* We don't fold IR::Members until after type checking.

* In particular, we need to know the type of the expression to create a constant with the appropriate width.

* But the type checker requires header stacks to have a compile-time known constant as their size.